Modules/_decimal/_decimal.c

```diff
@@ -5775,11 +5771,8 @@ static struct ssize_constmap ssize_constants [] = {
 struct int_constmap { const char *name; int val; };
 static struct int_constmap int_constants [] = {
     /* int constants */
-#ifdef EXTRA_FUNCTIONALITY
-    {"DECIMAL32", MPD_DECIMAL32},
-    {"DECIMAL64", MPD_DECIMAL64},
-    {"DECIMAL128", MPD_DECIMAL128},
```

Comment:
Why was it removed?

Reply:
Why we should keep a symbolic value for `XY` for `DECIMALXY`?

Reply:
Only for backwards compatibility. It woulnd't be nice to remove them without notice as they can exist in the wild.

Reply:
I don't think it's harmful to keep them. We should remove them in a follow-up PR with a proper deprecation notice (and possibly a deprecation period). They are still publicly named (couldn't they appear in `dir(decimal)` for instance?)

Reply:
> They are still publicly named (couldn't they appear in dir(decimal) for instance?)

Yes, with (undocumented) -DEXTRA_FUNCTIONALITY.  Given decision on https://github.com/python/cpython/pull/122003/files#r1928477811 - this should be considered as private.

Reply:
OTOH, considering that there was a NEWS entry already mentioning `EXTRA_FUNCTIONALITY` (that you wrote IIRC) last summer, I still think that we can be conservative. It doesn't hurt keeping those constants IMO. And if you really want to remove them, I would like this to be done in a separate PR with a proper deprecation notice (maybe no deprecation period as they are totally private, but at least we should mention that they were removed). Victor had sufficient surprises when removing private functions caused troubles, so let's try to be nice.

Reply:
Why not keeping this code under the hidden feature EXTRA_FUNCTIONALITY?

Reply:
> Why not keeping this code under the hidden feature EXTRA_FUNCTIONALITY?

Good idea, done.